src/Selenium2Library/locators/elementfinder.py

```diff
@@ -23,19 +42,33 @@ def __init__(self):
             'scLocator': self._find_by_sc_locator,
             'default': self._find_by_default
         }
-        self._strategies = NormalizedDict(initial=strategies, caseless=True, spaceless=True)
+        self._strategies = NormalizedDict(initial=strategies, caseless=True,
+                                          spaceless=True)
         self._default_strategies = list(strategies.keys())
+        self._key_attrs = {
+            None: ['@id', '@name'],
+            'a': [
+                '@id', '@name', '@href',
+                'normalize-space(descendant-or-self::text())'
+            ],
```

Comment:
A bit verbose formatting, especially compared to how the earlier entry is on one line only. #nitpick